src/libcrun/container.c

```diff
@@ -2724,7 +2744,7 @@ libcrun_container_run_internal (libcrun_container_t *container, libcrun_context_
       /* Do not open the notify socket here on "create".  "start" will take care of it.  */
       ret = get_notify_fd (context, container, &notify_socket, err);
       if (UNLIKELY (ret < 0))
-        return cleanup_watch (context, def, cgroup_status, pid, sync_socket, terminal_fd, err);
+        goto fail;
```

Comment:
Just a small question:  Is motivation behind behind this `goto` is to remove redundant `return` lines.

Reply:
The motion is that the function call is quite verbose.  With the goto, it is written only ib one place

Reply:
ah i see